src/EmbeddedSonarAnalyzer.props

```diff
@@ -5,7 +5,7 @@
   <PropertyGroup>
     <!-- Note: don't forget to upload the new packages to the package feed when changing 
```

Comment:
Maybe it's time to change the comment :P replace it with pointing to the xtranet